### ISSUE TITLE
When are you testing on all local models, why is it being called train accuracy and train loss?

### ISSUE THREAD

**Issue by harora** [AshwinRJ/Federated-Learning-PyTorch#13]
In federated_main.py file ,  once you average out the weights, you calculate accuracy on TEST set of each client. So why is that you're calling it train accuracy/loss? 

Anything I'm missing here?
